### PR TITLE
Update links in the Readme

### PR DESCRIPTION
**Pull request by skanwal**
- Fixed readme links
- Added support documents

### CODE REVIEW

**Comment by skanwal** [comment]
I need to fix the `Usage` part which is pointing to older version.

**Comment by skanwal** [comment]
@pdiakumis - this one's ready

File: inst/articles/workflow.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+* **Extract** expression level **information** and genome-based findings for genes located within detected CNVs (use ```--cn_loss ``` and ```--cn_gain ``` [arguments](README.md/#arguments) to define CN threshold values to classify genes within lost and gained regions)
+* **Ordered genes** by increasing (for genes within lost regions) or decreasing (for genes within gained regions) **[CN](https://github.com/umccr/umccrise/blob/master/workflow.md#somatic-cnv)** and then by decreasing absolute values representing difference between expression levels in the patient sample and the corresponding reference cohort
+
+## 4. Results annotation
+
+[WTS](./README.md#wts)- and/or [WGS](./README.md#wgs)-based results for the altered genes are collated with **knowledge** derived from in-house resources and public **databases** (listed below) to provide additional source of evidence for their significance, e.g. to flag variants with clinical significance or potential druggable targets.
+
+### Key cancer genes
+
+* [UMCCR key cancer genes set](https://github.com/vladsaveliev/NGS_Utils/blob/master/ngs_utils/reference_data/key_genes/make_umccr_cancer_genes.Rmd) build of off several sources:
```
Comment (pdiakumis) [inline inst/articles/workflow.md:179]: I'd point to the umccr fork - https://github.com/umccr/NGS_Utils/blob/master/ngs_utils/reference_data/key_genes/make_umccr_cancer_genes.Rmd

**Review by pdiakumis — APPROVED** [review]
Thanks, only a couple suggestions.

**Pull request merged**